web-app/src/app/screens/Analytics/GTFSFeatureAnalytics/index.tsx

```diff
@@ -0,0 +1,225 @@
+import { useState, useEffect, useMemo } from 'react';
+import { useLocation, useNavigate } from 'react-router-dom';
+import {
+  MaterialReactTable,
+  type MRT_ColumnDef,
+  type MRT_Cell,
+  useMaterialReactTable,
+} from 'material-react-table';
+import {
+  XAxis,
+  YAxis,
+  CartesianGrid,
+  Tooltip,
+  Legend,
+  ResponsiveContainer,
+  Brush,
+  Line,
+  LineChart,
+} from 'recharts';
+import Box from '@mui/material/Box';
+import { Typography, Button } from '@mui/material';
+import * as React from 'react';
+import { useTheme } from '@mui/material/styles';
+import { InfoOutlined, ListAltOutlined } from '@mui/icons-material';
+import { featureGroups, getGroupColor } from '../../../utils/analytics';
+import { type FeatureMetrics } from '../types';
+import { useRemoteConfig } from '../../../context/RemoteConfigProvider';
+
+export default function GTFSFeatureAnalytics(): React.ReactElement {
+  const navigateTo = useNavigate();
+  const { search } = useLocation();
+  const params = new URLSearchParams(search);
+  const featureName = params.get('featureName');
+  const [data, setData] = useState<FeatureMetrics[]>([]);
+  const [loading, setLoading] = useState(true);
+  const { config } = useRemoteConfig();
+
+  useEffect(() => {
+    const fetchData = async (): Promise<void> => {
+      try {
+        const response = await fetch(
+          `${config.gtfsMetricsBucketEndpoint}/features_metrics.json`,
+        );
+        if (!response.ok) {
+          throw new Error('Network response was not ok');
+        }
+        const fetchedData = await response.json();
+        const dataWithGroups = fetchedData.map((feature: FeatureMetrics) => ({
+          ...feature,
+          latest_feed_count: feature.feeds_count.slice(-1)[0],
+          feature_group: Object.keys(featureGroups).find((group) =>
+            featureGroups[group].includes(feature.feature),
+          ),
+        }));
+        setData(dataWithGroups);
+      } catch (error) {
+        console.error('Error fetching data:', error);
```

Comment:
[follow up ticket] basic error state

Reply:
Yes this is part of my todo: https://github.com/orgs/MobilityData/projects/82?pane=issue&itemId=76666507